test/fixtures/cookie-favicon.html

```diff
@@ -0,0 +1,37 @@
+<html>
+<head>
+</head>
+<body>
+
+<iframe id='test-frame' height=200 width=500></iframe>
+
+<script>
+var host = window.location + ''
+var parts = host.split('/')
+parts.splice(-1, 1)
+host = parts.join('/')
+
+function loadIco() {
+  var faviconLocation = host.replace('127.0.0.1', 'localhost') + '/cookie-favicon.ico'
```

Comment:
do you need to git add cookie-favicon.ico?

Reply:
nvm i see https://github.com/brave/browser-laptop/pull/14286/files#diff-6223bbd944b0287351445529f464dc1fR89